cmd/jujud/agent/machine_test.go

```diff
@@ -1365,6 +1366,20 @@ func (s *MachineSuite) TestReplicasetInitForNewController(c *gc.C) {
 	c.Assert(s.fakeEnsureMongo.InitiateCount, gc.Equals, 0)
 }
 
+func (s *MachineSuite) TestGetAuditLogConfig(c *gc.C) {
+	cfg := coretesting.FakeControllerConfig()
+	cfg["auditing-enabled"] = true
+	cfg["audit-log-exclude-methods"] = []interface{}{"Exclude.This"}
```

Comment:
why not []string{"Exclude.This"}

Reply:
See above.

Reply:
See above - this is what the coerce call returns when constructing the config.